docker/pypi/wmagent/init.sh

```diff
@@ -118,6 +119,15 @@ _copy_runtime() {
         echo "$FUNCNAME: Copying content from: $WMA_DEPLOY_DIR/etc/ to: $WMA_CONFIG_DIR/"
         cp -ra $WMA_DEPLOY_DIR/etc/ $WMA_CONFIG_DIR/
         echo $WMA_BUILD_ID > $wmaInitRuntime
+
+        # NOTE: If the sql database have been previously initialized, then the
+        #       current call of _copy_runtime has been due to a change of the
+        #       $WMA_TAG - either a patch version or a release candidate change
+        #       and we need to update the record at the database as well
+        _init_valid $wmaInitSqlDB && {
```

Comment:
I would suggest to make a new function and separate this action from the runtime action.

Reply:
Actually it must be inside the  `_copy_runtime` action, because we have to make  sure that once the runtime files have been copied we must update the database reflecting the actual flag. What I mean those two steps are sequential and dependent.